lms/static/scripts/frontend_apps/utils/fetch.ts

```diff
@@ -51,6 +53,10 @@ export function useFetch<T = unknown>(
   key: string | null,
   fetcher?: Fetcher<T>,
 ): FetchResult<T> {
+  // If an InitialLoadingContext is found, we want to notify about loading
+  // start/finish
+  const globalLoading = useContext(InitialLoadingContext);
```

Comment:
I added this here, because it's convenient. With this, any dashboard component using `useFetch` or `useAPIFetch` will automatically notify the global indicator if the context was not `null`.

Any other usages inside the other LMS modes should not be affected.